experiments/20161104-pol-assay-high-throughput/20161117-second-run.md

```diff
@@ -17,6 +17,30 @@ For one replicate:
 6. Add 32 ul of appropriate transfection mixture to each well.
 
 
-2016-11-17 Notes:
+**2016-11-17 Notes:**
 - 2.7 million cells/mL -> added 3mL of DMEM to the 1mL solution, seeded 4ul of diluted solution for 30000 cells/well.
 - **Dilution:** Seeded three wells each of 1:1, 1:2, 1:4, and 1:8 dilutions. 1:1 contained 30000 cells.
+
+**2016-11-19 Transfection Notes:**
+- Uneven growth of cells in wells
+- The first sample of Vic 1:1 dilution does not appear to contain any cells.
+- The second row of library replicates appeared to be sparse on cells as well
+
+Reagents:
+- Made 1:8 dilutions of all mutant library PB2 plasmids. Standardized to 4ul PB2 plasmid in each transfection solution
+
+### Plate layout
+|-----|-----|-----|-----|-----|-----|-----|-----|------|-----|-----|------|
+| 1:1 | 1:1 | 1:1 | 1:2 | 1:2 | 1:2 | 1:4 | 1:4 | 1:4  | 1:8 | 1:8 | 1:8  |
+| L17 | L18 | L19 | L20 | L22 | L23 | L24 | L26 | Mock | PHH | PCD | Mock |
+| L17 | L18 | L19 | L20 | L22 | L23 | L24 | L26 | Mock | PHH | PCD | Mock |
+
+- First row samples were all transfected with Vic PB2 plasmids
+-
+
+**2016-11-22 Luminescence Testing Notes:**
+- Accidentally left luciferase supernatant samples in 25C for 15 hours, but signal was still picked up during luminescence scanning. Only the first two rows of samples were tested.
+
+### Tips for improvement from Kim
+- Grow cells in 100ul DMEM, so that they settle more quickly and will be less likely to clump in media
+- Mix cells in DMEM before seeding, and mix thoroughly in Falcon tube and in trough
```

Comment:
Great stuff picking up these tips! Can you ensure they are reflected in the protocol as well?